test/secret/secret_s1_test.go

```diff
@@ -181,6 +186,10 @@ var _ = Describe("Secret Test for SVA S1", func() {
 
 			// Verify Secrets on ServerConf on Pod
 			testenv.VerifySplunkServerConfSecrets(deployment, testenvInstance, verificationPods, secretStruct.Data, false)
+
+			// Verify Secrets via api access on Pod
+			secretStruct, err = testenv.GetSecretStruct(deployment, testenvInstance.GetName(), namespaceScopedSecretName)
+			testenv.VerifySplunkSecretViaAPI(deployment, testenvInstance, verificationPods, secretStruct.Data)
```

Comment:
This looks like a positive match? However we need a negative verification here ?